primeqa/mitqa/mitqa_component.py

```diff
@@ -0,0 +1,117 @@
+import json
+from transformers import (
+    HfArgumentParser,
+    TrainingArguments,
+)
+from primeqa.mitqa.utils.model_utils.row_retriever_MITQA import RowRetriever
+from primeqa.mitqa.utils.model_utils.reranker import re_rank_ae_output
+from primeqa.mitqa.utils.link_predictor import predict_link_for_tables,train_link_generator
+from primeqa.mitqa.utils.model_utils.table_retriever import train_table_retriever,predict_table_retriever
+from primeqa.mitqa.utils.model_utils.process_row_retriever_output import preprocess_data_using_row_retrieval_scores,create_dataset_for_answer_extractor
+from primeqa.mitqa.utils.model_utils.answer_extractor_multi_Answer import train_ae,predict_ae
+from primeqa.mitqa.processors.preprocessors.preprocess_raw_data import preprocess_data,load_st_model
+import logging
+import torch
+import os
+import sys
+from primeqa.mitqa.utils.arguments_utils import HybridQAArguments,LinkPredictorArguments, RRArguments,AEArguments
+from primeqa.components.base import Component
+from primeqa.mitqa.metrics.evaluate_ottqa import get_em_and_f1_ottqa
+from primeqa.mitqa.metrics.evaluate import get_em_and_f1_hybridqa
```

Comment:
Sort the imports? E.g. from more general (os) to specific (torch) to local (primeqa).